--- steem-0.18.103.old/steem/cli.py	2017-09-19 17:37:07.000000000 +0200
+++ steem-0.18.103.new/steem/cli.py	2018-02-13 19:48:51.197607400 +0100
@@ -16,7 +16,7 @@
 from .block import Block
 from .blockchain import Blockchain
 from .dex import Dex
-from .instance import shared_steemd_instance
+from .instance import shared_steemd_instance, set_shared_steemd_instance
 from .post import Post
 from .utils import construct_identifier, strfage
 from .witness import Witness
@@ -47,7 +47,7 @@
         '--node',
         type=str,
         default=configStorage["node"],
-        help='URL for public Steem API (default: "https://steemd.steemit.com")'
+        help='URL for public Steem API (default: "https://api.steemit.com")'
     )
 
     parser.add_argument(
@@ -929,7 +929,7 @@
 
     # initialize STEEM instance
     options = {
-        "node": args.node,
+        "nodes": [args.node] if args.node else None,
         "unsigned": args.unsigned,
         "expires": args.expires
     }
@@ -939,6 +939,7 @@
         options.update({"wif": []})
 
     steem = stm.Steem(no_broadcast=args.no_broadcast, **options)
+    set_shared_steemd_instance(steem)
 
     if args.command == "set":
         if (args.key in ["default_account"] and
