--- steem-0.18.103.old/steem/steemd.py	2017-09-19 17:37:07.000000000 +0200
+++ steem-0.18.103.new/steem/steemd.py	2018-02-13 18:45:49.942026700 +0100
@@ -50,7 +50,7 @@
 
     def __init__(self, nodes=None, **kwargs):
         if not nodes:
-            nodes = get_config_node_list() or ['https://steemd.steemit.com']
+            nodes = get_config_node_list() or ['https://api.steemit.com']
         
         super(Steemd, self).__init__(nodes, **kwargs)
 
